be/src/exec/vectorized/chunks_sorter.cpp

```diff
@@ -50,7 +50,7 @@ void DataSegment::init(const std::vector<ExprContext*>* sort_exprs, const ChunkP
 Status DataSegment::get_filter_array(std::vector<DataSegment>& data_segments, size_t rows_to_sort,
                                      std::vector<std::vector<uint8_t>>& filter_array,
                                      const std::vector<int>& sort_order_flags, const std::vector<int>& null_first_flags,
-                                     uint32_t& least_num, uint32_t& middle_num) {
+                                     uint32_t& before_num, uint32_t& in_num) {
```

Comment:
The name of before_num and in_num are not clear.

Reply:
The name is compatible with `DataSegment::BEFORE_LAST_RESULT` and `DataSegment::IN_LAST_RESULT`, maybe we need to change them all